collector/nodes.go

```diff
@@ -1362,6 +1362,20 @@ func NewNodes(logger log.Logger, client *http.Client, url *url.URL, all bool, no
 					return append(defaultNodeLabelValues(cluster, node), "mapped")
 				},
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "jvm", "uptime"),
```

Comment:
This should have the `_seconds` suffix to match prometheus naming standards https://prometheus.io/docs/practices/naming/